crates/op-rbuilder/src/tests/framework/txs.rs

```diff
@@ -118,10 +142,31 @@ impl TransactionBuilder {
     }
 
     pub async fn send(self) -> eyre::Result<PendingTransactionBuilder<Optimism>> {
+        let bundle_opts = self.bundle_opts.clone();
         let provider = self.provider.clone();
         let transaction = self.build().await;
+        let transaction_encoded = transaction.encoded_2718();
+
+        if let Some(bundle_opts) = bundle_opts {
+            // Send the transaction as a bundle with the bundle options
+            let bundle = Bundle {
+                transactions: vec![transaction_encoded.into()],
+                block_number_max: bundle_opts.block_number_max,
+            };
+
+            let tx_hash = provider
+                .client()
+                .request("eth_sendBundle", (bundle,))
```

Comment:
Do we want to make this more explicit and make the `TransactionBuilder` build a `Bundle` type or have `BundleBuilder` type altogether? Not entirely sure if this preserves separation of concerns.